src/app/features/forecast/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="sidebar">
+  <div class="sidebar__header">
+    <app-button
+      label=" Search for places"
+      [buttonStyles]="buttonStyles"
+      [type]="'button'"
+      (handleClick)="showMessage()"
+    ></app-button>
+    <app-badge
+      [badgeStyles]="badgeStyles"
```

Comment:
better define several css classes, provide button type and choose one of defined css classes